RecoMET/METFilters/python/ecalEEBadCalibSep2017ListFilter_cfi.py

```diff
@@ -0,0 +1,28 @@
+import FWCore.ParameterSet.Config as cms
+ 
+ecalBadCalibSep2017ListFilter = cms.EDFilter(
```

Comment:
names like this one look very strange in a few years time - could we pick one that describes what physically the module is doing? [as there is no run number filter defined]

Reply:
I think that Sep2017 filter is much less strange because it's a rather explicit name tag.
My assumption is that the group maintains documentation of what these lists select.
A more specific name here (instead of "a list made in Sep2017") would be derived from "list of IDs with bad calibrations in GTs used for 2016 legacy rereco and in 92X prompt".
I'm not sure if this applies anymore to even the early rereco made with 12Sep2017
and I expect (but I have no explicit confirmation) that these are already fixed in the default GT of 94X.